librz/main/rizin.c

```diff
@@ -402,8 +402,8 @@ RZ_API int rz_main_rizin(int argc, const char **argv) {
 		free(sysdbg);
 	}
 	if (argc < 2) {
-		LISTS_FREE();
-		return main_help(1);
+		//LISTS_FREE();
```

Comment:
??

Reply:
> I've added to the comment0 some things to try as tests, mentioned in the referenced issues. I'm not sure yet if automatic tests were added for those, but they should be tested manually before approving this.

Thans for your suggestion ,  I manually test use the comment0 test and added the result to the comment0 .